operator/controller/src/main/java/io/apicurio/registry/operator/utils/Utils.java

```diff
@@ -8,4 +14,42 @@ private Utils() {
     public static boolean isBlank(String value) {
         return value == null || value.isBlank();
     }
+
+    /**
+     * Merge source map into target map, NOT overriding any entries with the same key.
+     *
+     * @param target must not be null
+     */
+    public static <V> void mergeNotOverride(Map<String, V> target, Map<String, V> source) {
```

Comment:
As mentioned, I'm not in favor of introducing those levels of indirection, they are making the code harder to read with very limited in terms of saved locs

Reply:
Can it be done in a followup PR from you?